### ISSUE TITLE
Document `/` in the component path

### ISSUE THREAD

**Issue by mixonic** [rwjblue/ember-module-migrator#60]
Document that if you use `{{component someVar}}` and `someVar` is `"foo/bar-baz"` the lookup should be changed to not use a `/`.

Additionally I'm not sure what components invoked as `{{foo/bar-baz}}` end up as.

Refs: https://github.com/rwjblue/ember-module-migrator/issues/53

/cc @billybonks

**Comment by mixonic** [comment]
Added docs, still waiting on some research into what the static invocation does.

**Comment by mixonic** [comment]
Ah it moves them incorrectly. I'll open a new issue regarding this and close this one.

**Comment by billybonks** [comment]
well, it moves them to components global because there is no way to know about dynamic components unless you parse the code.
